lib/utils/constants.dart

```diff
@@ -15,3 +15,7 @@ const double imageHeight = 130.0;
 const double gridContentSpacing = 20.0;
 const pokemonImageUrlPath =
     'https://raw.githubusercontent.com/PokeAPI/sprites/master/sprites/pokemon/other/home/[index].png';
+
+const String errorMessage = 'Something went wrong!';
+const String emptyListErrorMessage = 'No Pokemons Available';
+final scaffoldKey = GlobalKey<ScaffoldMessengerState>();
```

Comment:
no need for this, just directly place this as a constant above ur `PokemonOverviewPage` but don't forget to make it private 🙂